easybuild/easyconfigs/s/scArches/scArches-0.6.1-foss-2023a.eb

```diff
@@ -0,0 +1,88 @@
+easyblock = 'PythonBundle'
+
+name = 'scArches'
+version = '0.6.1'
+
+homepage = 'https://github.com/theislab/scarches'
+description = """Single-cell architecture surgery (scArches) is a package for reference-based analysis of single-cell
+ data."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('scikit-build-core', '0.5.0')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('matplotlib', '3.7.2'),
+    ('anndata', '0.10.5.post1'),
+    ('h5py', '3.9.0'),
+    ('leidenalg', '0.10.2'),
+    ('scanpy', '1.9.8'),
+    ('scikit-learn', '1.3.1'),
+    ('scvi-tools', '1.1.2'),
+    ('PyTorch', '2.1.2'),
+    ('tqdm', '4.66.1'),
+    ('IPython', '8.14.0'),
+    ('pkgconfig', '1.5.5', '-python'),
+    ('PyZMQ', '25.1.1'),
+    ('tornado', '6.3.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('sphinx_basic_ng', '1.0.0b2', {
+        'checksums': ['9ec55a47c90c8c002b5960c57492ec3021f5193cb26cebc2dc4ea226848651c9'],
+    }),
+    ('nodeenv', '1.9.0', {
+        'checksums': ['07f144e90dae547bf0d4ee8da0ee42664a42a04e02ed68e06324348dafe4bdb1'],
+    }),
+    ('sphinx-theme-builder', '0.2.0b2', {
```

Comment:
You'll need to add `pyproject-metadata` as extension here too, because `sphinx-theme-builder` requires it, see failing `pip check`